stats/measures.go

```diff
@@ -27,4 +27,10 @@ var (
 		stats.UnitSeconds)
 	// HTTPRequests measures the count of HTTP requests.
 	HTTPRequests = stats.Int64("HTTPRequests", "Measures the count of HTTP requests", stats.UnitDimensionless)
+	// ShellParseErrors measures the count of errors when attempting to parse shell code.
+	ShellParseErrors = stats.Int64(
```

Comment:
Can we use existing `CheckErrors` to record these? You'll need to add a new error here: https://github.com/ossf/scorecard/blob/main/errors/public.go#L23 and update the GetName fn - https://github.com/ossf/scorecard/blob/main/errors/public.go#L44

Reply:
I considered that at first, but I wasn't sure if it would skew existing error count metrics. I've gone ahead and updated this PR to use `CheckErrors` since you've given me the confidence to do so.